From 595edde30ccad4f83a5693f7ca77c68af07f0569 Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@toradex.com>
Date: Thu, 16 Apr 2020 11:03:54 +0300
Subject: [PATCH 1/3] wireless: make wireless_ext and wext_priv storable

The config option WIRELESS_EXT and WEXT_PRIV use to build external
drivers, such as backports. Make these options storable in config file
to be able to set them independently.

Related-to: ELB-2388
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@toradex.com>
---

 net/wireless/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/wireless/Kconfig b/net/wireless/Kconfig
index 6c606120abfe..9fa6e6d34689 100644
--- a/net/wireless/Kconfig
+++ b/net/wireless/Kconfig
@@ -1,5 +1,5 @@
 config WIRELESS_EXT
-	bool
+	bool "WIRELESS_EXT - wireless core extensions"
 
 config WEXT_CORE
 	def_bool y
@@ -14,7 +14,7 @@ config WEXT_SPY
 	bool
 
 config WEXT_PRIV
-	bool
+	bool "WEXT_PRIV - Wireless Extensions priv API"
 
 config CFG80211
 	tristate "cfg80211 - wireless configuration API"
-- 
2.20.1

